starter-code/src/App.js

```diff
@@ -1,18 +1,135 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import "./App.css";
+import "bulma/css/bulma.css";
+import foods from "./foods.json";
+import FoodBox from "./components/FoodBox";
+import AddFoodForm from "./components/AddFoodForm";
+import SearchBar from "./components/SearchBar";
+import TodaysFood from "./components/TodaysFood";
 
 class App extends Component {
+  state = {
+    isOnClick: false,
+    foods: [...foods],
+    currentList: [],
+    todaysFood: [],
+    inputNumber: 0,
+    totalCalorie: 0
+  };
+
+  addFood = newFood => {
```

Comment:
be careful you have 2 addFood function and the second is over-writting the first one, so your form is not working anymore.